pkg/features/kube_features.go

```diff
@@ -98,7 +94,6 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	ExternalTrafficLocalOnly:                    {Default: true, PreRelease: utilfeature.GA},
```

Comment:
Yeah, that's probably not OK.  @kubernetes/sig-cluster-lifecycle-pr-reviews -- comments?